runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerPlugin.java

```diff
@@ -142,6 +142,11 @@ protected void startUp() throws Exception
 	{
 		xpPanel = new XpPanel(this, xpTrackerConfig, client, skillIconManager);
 
+		for (Skill skill : Skill.values())
```

Comment:
this should just be `xpState.reset()`

Reply:
`xpState.reset()` doesn't initialize the skills to the current xp and the issue persists.

Reply:
What happens if you toggle the plugin while logged out?

Reply:
Nothing special from what I can tell.

Reply:
I believe it doesn't zero the skill xps out if you logout. Is that bad?

Reply:
No, because you might want to continue your skilling session after a break.